bindings/tbdex_uniffi/src/tbdex.udl

```diff
@@ -286,19 +285,18 @@ dictionary ServiceData {
   sequence<string> service_endpoint;
 };
 
+[Error]
 interface Web5RustCoreError {
-  string error_type();
-  string variant();
-  string message();
+  Error(string type, string variant, string msg);
 };
 
-[Trait]
+[Trait, WithForeign]
 interface Signer {
   [Throws=Web5RustCoreError]
-  bytes sign([ByRef] sequence<u8> payload);
+  bytes sign(bytes payload);
 };
 
-[Trait]
+[Trait, WithForeign]
```

Comment:
We are going to import this from web5-rs and delete this soon (tm) right?

Reply:
That's right, but there is a slight nuance with the way UniFFI bindings will work but you'll see in due time

But yes, the main thing is that right now in the tbdex KT project we have `tbdex.sdk.web5` and that package needs to go away sooner rather than later